heat/core/operations.py

```diff
@@ -519,10 +527,11 @@ def __reduce_op(x, partial_op, reduction_op, axis, out):
 
     if axis is None:
         partial = partial_op(x._tensor__array).reshape((1,))
-        output_shape = (1,)
+        output_shape = (1,) if keepdim else ()
     else:
         partial = partial_op(x._tensor__array, dim=axis, keepdim=True)
```

Comment:
should this be changed to keepdim=keepdim?

Reply:
No, my understanding is that we definitely want to keep the dimension until after the MPI collective has run.

Reply:
okay sounds good